build_tools/update_environments_and_lock_files.py

```diff
@@ -187,7 +187,7 @@ def remove_from(alist, to_remove):
         "build_name": "pylatest_pip_openblas_pandas",
         "folder": "build_tools/azure",
         "platform": "linux-64",
-        "channel": "defaults",
+        "channel": "default",
```

Comment:
Why this change? The correct channel name is `defaults` is it not?

```

Reply:
The default channel (that the `conda` executable uses) is `defaults` (with S). I think the reason you get conda-forge packages with your command is that the default channel for `mamba` is conda-forge (and adding a channel that doesn't exist (`default`) is apparently not an error).